Tensile/LibraryIO.py

```diff
@@ -410,3 +412,38 @@ def createLibraryLogicList(schedulePrefix, architectureName, deviceNames, logicT
 
     data.append(logicTuple[7])
     return data
+
+
+def initAsmCapsCache(cacheFile: str) -> Optional[dict]:
+    """
+    If requested and exists, reads an asm cache from disk.
```

Comment:
I've never loved that we shorten "assembler capabilities" to AsmCaps, as (imo) it obscures the target. However, I do understand that it can be overly verbose to type it all out. Perhaps updating this doc-string to say,

> If requested and exists, reads the cached assembler capabilities (asm caps) from disk.

Will help provide clarity to Tensile newcomers?

Reply:
Fixed